homeassistant/components/template/fan.py

```diff
@@ -303,7 +301,6 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
 
         self._state = STATE_ON
         self._preset_mode = preset_mode
-        self._percentage = None
```

Comment:
Were these changes to the template fan intended?

Reply:
No :( That was another PR I was preparing... Sorry about that!